front-end/src/app/app.module.ts

```diff
@@ -1,5 +1,8 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ServiceWorkerModule } from '@angular/service-worker';
+
```

Comment:
Line

Reply:
Because it separates external libs from internals one.